index.html

```diff
@@ -9,11 +9,102 @@
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
-<body>
+    <body>
+        <!-- HEADER -->
+        <header class="header">
+            <div>
+                <img class="header-image" src="img/karma-logo.svg" alt="karma logo"/>
+            </div> 
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+            <!-- HEADER NAVIGATION -->
+            <nav>
+                <ul class="nav-list">
+                    <li class="nav-item">
+                        <a class="nav-link" href="#"><strong>Meet Karma</strong></a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#">How it Works</a> 
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#">Store</a> 
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#">Blog</a> 
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#">Help</a> 
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#">Login</a> 
+                    </li>
+                </ul>
+            </nav>
+        </header>
 
-</body>
+        <!-- MAIN -->
+        <main>
+            <!-- TOP HALF OF PAGE -->
+            <div class="page-top-half">
+                <img class="background-image" src="img/first-background.jpg" alt="background-image"/>
+
+                <!-- TEXT/BUTTON ON BACKGROUND IMAGE -->
+                <h1 class="large-text">Introducing Karma</h1>
+                <p class="large-text">Bring WiFi with you, everywhere you go.</p>
```

Comment:
Maybe this would be better as an `h2` element instead of `p` as it's more like a sub-heading than a paragraph